keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -314,8 +321,7 @@ No change from existing behavior of the Job controller.
 
 ###### What are other known failure modes?
 
-- When the cluster has apiservers with skewed versions, the `Job.status.ready`
-  might remain zero.
+No.
```

Comment:
I'm not actually aware of any failure modes currently. I'm also not sure why the field would be zero if the API server version is skewed. I assume, for older API servers, which don't know the field we would have `nil`. @alculquicondor for insights.